packages/askui-nodejs/src/execution/inference-client.ts

```diff
@@ -75,12 +81,18 @@ export class InferenceClient {
     const resizedImage = await this.resizeIfNeeded(customElements, image);
     const response = await this.httpClient.post<InferenceResponseBody>(
       this.urls.inference,
-      {
-        customElements,
-        image: resizedImage.base64Image,
+      this.urls.inference.endsWith('predict') ? {
+        // V4 only supports the base64 image and not the metadata of the base64 string
+        image: resizedImage.base64Image?.split(',')[1],
```

Comment:
Should we move this task to the backend, like we do [here](https://github.com/askui/askui-inference/blob/dbe7d649e72a2681cf19dd066245ee3f1a31ed1d/controlyourui/helper.py#L24)?